packages/rulesets/generated/spectral/az-dataplane.js

```diff
@@ -2190,6 +2202,8 @@ const ruleset = {
             description: "All schema properties should have a description.",
             message: "Property should have a description.",
             severity: "warn",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
```suggestion
            disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-core/bad-record-type' rule.",
```

I believe this is the same rule

Reply:
No this is the documentation-required rule.

Reply:
hhm copy paste issue probably